interfaces/schema/postSchema.ts

```diff
@@ -10,6 +10,9 @@ export const postSchema = z.object({
       required_error: 'Description is required',
     })
     .min(15),
+  content: z.string({
+    required_error: 'Content is required',
```

Comment:
@titusjoyson @rohitdasu do we add any minimum character limit

Reply:
We should add imo

Reply:
Since its a markdown and assuming it will be the content of the project details page, I don't think setting less than 250 will work.

any suggestions on the min value?

Reply:
I think it will work

Reply:
Any update?